db/migrate/20240916101011_remove_teacher_reference_number_from_job_applications.rb

```diff
@@ -0,0 +1,6 @@
+
+gclass RemoveTeacherReferenceNumberFromJobApplications < ActiveRecord::Migration[7.1]
```

Comment:
I suspect this is breaking all the CI runs. 
```suggestion
class RemoveTeacherReferenceNumberFromJobApplications < ActiveRecord::Migration[7.1]
```